### PR TITLE
Adding support to allow customizing the default controls along with s…

### PR DESCRIPTION
**Pull request by frodare**
…ome minor refactors.

### CODE REVIEW

**Review by mrprim — APPROVED** [review]

File: src/components/DefaultControls/PageSizer.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  return (
+    <select value={pageSize} onChange={handlePageSizeChange}>
+      {pageSizes.map(size => <option key={size} value={size}>{pageSizeLableGenerator(size)}</option>)}
+    </select>
+  )
+}
+
+PageSizer.propTypes = {
+  pageSizes: propTypes.arrayOf(propTypes.number),
+  pageSizeLableGenerator: propTypes.func
```
Comment (zeriley) [inline src/components/DefaultControls/PageSizer.js:25]: Label

File: src/components/DefaultControls/index.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+import PageSizer from './PageSizer'
+import Status from './Status'
+
+const DefaultControls = ({ className, pageSizes, pageSizeLableGenerator, statusMessageGenerator }) => (
+  <div className={'manifest-controls' + (className ? ' ' + className : '')}>
+    <PageSizer pageSizes={pageSizes} pageSizeLableGenerator={pageSizeLableGenerator} />
+    <Status statusMessageGenerator={statusMessageGenerator} />
+    <Pager />
+  </div>
+)
```
Comment (zeriley) [inline src/components/DefaultControls/index.js]: Eww, unnecessary parens

File: src/hooks/useStatus.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import useManifest from './useManifest'
+
+const determineFirstOnPage = (page, pageSize) => Math.max(page * pageSize, 0)
```
Comment (zeriley) [inline src/hooks/useStatus.js:3]: I feel like this method should just add 1, rather than adding 1 to it everywhere it's used

**Review by zeriley — APPROVED** [review]

**Review by zeriley — CHANGES_REQUESTED** [review]
I do want the label typo fixed.  The others don't need to be done, though.

File: src/components/DefaultControls/PageSizer.js (hunk 1 above)
Comment (frodare) [reply src/components/DefaultControls/PageSizer.js:25]: Arg, meant to fix that

**Review by zeriley — APPROVED** [review]

**Pull request merged**